src/mjolnir/valhalla_convert_transit.cc

```diff
@@ -1,1281 +1,7 @@
-
-  // A place to hold worker threads and their results
-  std::vector<std::shared_ptr<std::thread>> threads(thread_count);
-
-  // An atomic object we can use to do the synchronization
-  std::mutex lock;
-
-  // A place to hold the results of those threads (exceptions, stats)
-  std::list<std::promise<builder_stats>> results;
-
-  // Start the threads, divvy up the work
-  LOG_INFO("Adding " + std::to_string(all_tiles.size()) + " transit tiles to the transit graph...");
-  size_t floor = all_tiles.size() / threads.size();
-  size_t at_ceiling = all_tiles.size() - (threads.size() * floor);
-  std::unordered_set<GraphId>::const_iterator tile_start, tile_end = all_tiles.begin();
-
-  // Atomically pass around stats info
-  for (size_t i = 0; i < threads.size(); ++i) {
-    // Figure out how many this thread will work on (either ceiling or floor)
-    size_t tile_count = (i < at_ceiling ? floor + 1 : floor);
-    // Where the range begins
-    tile_start = tile_end;
-    // Where the range ends
-    std::advance(tile_end, tile_count);
-    // Make the thread
-    results.emplace_back();
-    threads[i].reset(new std::thread(build_tiles, std::cref(pt.get_child("mjolnir")), std::ref(lock),
-                                     std::cref(all_tiles), tile_start, tile_end,
-                                     std::ref(results.back())));
-  }
-
-  // Wait for them to finish up their work
-  for (auto& thread : threads) {
-    thread->join();
-  }
-
-  // Check all of the outcomes, to see about maximum density (km/km2)
-  builder_stats stats{};
-  uint32_t total_no_dir_edge_count = 0;
-  uint32_t total_dep_count = 0;
-  uint32_t total_midnight_dep_count = 0;
-
-  for (auto& result : results) {
-    // If something bad went down this will rethrow it
-    try {
-      auto thread_stats = result.get_future().get();
-      stats(thread_stats);
-      total_no_dir_edge_count += stats.no_dir_edge_count;
-      total_dep_count += stats.dep_count;
-      total_midnight_dep_count += stats.midnight_dep_count;
-    } catch (std::exception& e) {
-      // TODO: throw further up the chain?
-    }
-  }
-
-  if (total_no_dir_edge_count) {
-    LOG_ERROR("There were " + std::to_string(total_no_dir_edge_count) +
-              " nodes with no directed edges");
-  }
-
-  if (total_dep_count) {
-    float percent =
-        static_cast<float>(total_midnight_dep_count) / static_cast<float>(total_dep_count);
-    percent *= 100;
-
-    LOG_INFO("There were " + std::to_string(total_dep_count) + " departures and " +
-             std::to_string(total_midnight_dep_count) +
-             " midnight departures were added: " + std::to_string(percent) + "% increase.");
-  }
-
-  auto t2 = std::chrono::high_resolution_clock::now();
-  uint32_t secs = std::chrono::duration_cast<std::chrono::seconds>(t2 - t1).count();
-  LOG_INFO("Finished building transit network - took " + std::to_string(secs) + " secs");
-}
-
 int main(int argc, char** argv) {
   if (argc < 2) {
```

Comment:
if we did the other executable with cxxopts we should probably do this one too. thankfully there are only what 2 arguments to parse, the config and a test file. should be quick!